apps/explorer/src/components/validator/ValidatorStats.tsx

```diff
@@ -116,14 +115,24 @@ export function ValidatorStats({
                         <Stats
                             label="Last Epoch"
                             tooltip="Coming soon"
-                            unavailable
-                        />
+                            unavailable={+lastEpoch <= 0}
```

Comment:
Can this be less than 0? And if last epoch was 0 why we don't show 0?

Reply:
You are right, I'll remove it